packages/pigeon/test/dart_generator_test.dart

```diff
@@ -144,7 +144,7 @@ void main() {
     final StringBuffer sink = StringBuffer();
     generateDart(DartOptions(), root, sink);
     final String code = sink.toString();
-    expect(code, isNot(matches('=.*doSomething')));
+    expect(code, isNot(matches('[^!]=.*doSomething')));
```

Comment:
minor nit: some comment would be helpful. It took me a while to figure out what it was checking :)

Reply:
fair, me too...

Reply:
done